extensions/vscode/webviews/homeView/src/stores/home.ts

```diff
@@ -57,6 +60,32 @@ export const useHomeStore = defineStore("home", () => {
     },
   );
 
+  watch(
+    [selectedContentRecord, selectedConfiguration],
+    ([contentRecord, config], [prevContentRecord, prevConfig]) => {
+      const result = new Map<string, string | undefined>();
+
+      if (config === undefined || isConfigurationError(config)) {
+        return result;
```

Comment:
return out of a watch? Does this do something that I'm not aware of?

Reply:
Oops this should be setting `secrets.value` then bailing out.

Reply:
Fixed in https://github.com/posit-dev/publisher/pull/2288/commits/08bccd44ed6aef1313ff2db0aaaf41d8bda1e4f0